include/circular_buffer.hpp

```diff
@@ -277,6 +332,16 @@ class circular_buffer
             }
         }
 
+        [[nodiscard]] constexpr auto get_alt(std::size_t n_slots_to_claim) noexcept -> std::pair<std::span<U>, WriterToken> {
```

Comment:
Maybe a more communicative name than `get_alt`?

Reply:
yes, this function will eventually replace the 'get(..)' function.